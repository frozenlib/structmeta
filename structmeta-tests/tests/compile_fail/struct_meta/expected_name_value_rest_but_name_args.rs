#[structmeta_tests::attr_rest_name_value(value(10))]
fn my_func() {}

fn main() {}
