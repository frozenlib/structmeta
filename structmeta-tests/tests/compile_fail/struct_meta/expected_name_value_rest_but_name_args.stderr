error: expected `value = ...`, found `value`(...)
 --> $DIR/expected_name_value_rest_but_name_args.rs:1:42
  |
1 | #[structmeta_tests::attr_rest_name_value(value(10))]
  |                                          ^^^^^
